### PR TITLE
sci-physics/bullet: version bump to 2.89

### PR DESCRIPTION
**Pull request by rossbridger**
This PR is a follow-up of https://github.com/gentoo/gentoo/pull/17226

Package-Manager: Portage-3.0.4, Repoman-3.0.1
Signed-off-by: Ross Charles Campbell <rossbridger.cc@gmail.com>

### CODE REVIEW

**Comment by rossbridger** [comment]
ping @SoapGentoo

File: sci-physics/bullet/bullet-2.89.ebuild [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+# Building / linking of third Party library BussIK does not work out of the box
+RESTRICT="test"
+
+S="${WORKDIR}/${PN}3-${PV}"
+
+src_prepare() {
+	if use openmp ; then
+		tc-has-openmp || die "Please switch to an openmp compatible compiler"
+	fi
```
Comment (SoapGentoo) [inline sci-physics/bullet/bullet-2.89.ebuild:45]: remove this, and instead add two phase functions:
```
pkg_pretend() {
	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
}

pkg_setup() {
	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
}
```

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sci-physics/bullet/bullet-2.89.ebuild (hunk 1 above)
Comment (SoapGentoo) [reply sci-physics/bullet/bullet-2.89.ebuild:45]: Also, don't forget to inherit `toolchain-funcs` for this

Comment (rossbridger) [reply sci-physics/bullet/bullet-2.89.ebuild:45]: Looks simple enough, fixing it right away

**Comment by rossbridger** [comment]
Is there anything else that requires further testing?

**Comment by rossbridger** [comment]
@SoapGentoo can we merge this PR now?

**Pull request closed without merging**